### DISCUSSION TITLE
Add a feature so you can switch channels on the same email address.

### DISCUSSION THREAD

**Discussion by Sparkkkkie** [ThioJoe/YT-Spammer-Purge#562] [Ideas]
So, my email address on youtube has more than one youtube accounts, 3 to be exact, and the problem is that i cant switch to a different account on that email address, could you add that?

**Comment by Masamune3210** [comment]
Can you not just supply the channel id?

**Reply by Sparkkkkie** [reply]
Well, i know that, but some people just dont know how to edit that or create the config, and so it would be easier to add that the script auto detects if you have multiple channels, if so if you would like to switch.

**Comment by SeqBra** [comment]
I have the same on my account, I might look into this later🤷

**Comment by KendallDoesCoding** [comment]
Just make your other account mod on the other channel?


You can make 2 directories

first directory: .exe file, client_secrets.json file of first account
second directory: .exe file, client_secrets.json file of second account

Depending on the channel, open the exe file respectively, that should work

**Reply by Sparkkkkie** [reply]
Well, if i do that it runs in mod only mode remember?

**Reply by KendallDoesCoding** [reply]
If you make a client_secrets.json in the second yt channel email it should work right? Unless the channel is a brand account
